src/App/Pages/Settings/SettingsPage/SettingsPage.xaml.cs

```diff
@@ -45,6 +46,17 @@ protected override bool OnBackButtonPressed()
             return base.OnBackButtonPressed();
         }
 
+        void ActivateTimePicker(object sender, EventArgs args)
+        {
+            var stackLayout = (ExtendedStackLayout)sender;
+            SettingsPageListItem item = (SettingsPageListItem)stackLayout.BindingContext;
+            if (item.ShowTimeInput)
+            {
+                var timePicker = stackLayout.Children.Where(x => x is TimePicker).FirstOrDefault();
+                ((TimePicker)timePicker).Focus();
```

Comment:
Given that `.FirstOrDefault()`, we should probably check for null before executing this just to be safe.